app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @user = current_user
+    @like = Like.new(author: @user, post: @post)
+    @like.save
+
+    flash[:success] = 'Liked'
+    redirect_to user_post_path(@user, @post)
```

Comment:
- [ ] It will be great to ensure you create the success and fail paths. Just like you have done in your other controllers, if the like is saved, do something, else do something else. Hint ⬇️ 
```ruby
 respond_to do |format|
        if @like.save
          format.html { redirect_to @post }
        else
          format.html { redirect_to @post }
        end
      end
```